ipaserver/install/server/replicainstall.py

```diff
@@ -904,7 +904,11 @@ def promote_check(installer):
                            "certificate")
 
     installutils.verify_fqdn(config.host_name, options.no_host_dns)
-    installutils.verify_fqdn(config.master_host_name, options.no_host_dns)
+    # Inside the container environment master's IP address does not
+    # resolve to its name. See bz #1364139  z
```

Comment:
Might be better to refer to https://pagure.io/freeipa/issue/6210 As the BZ is not available for most people. Also, the z at the end should not be here.